src/modules/commander/Commander.cpp

```diff
@@ -2140,6 +2243,13 @@ Commander::run()
 				}
 			}
 
+			if (_manual_control.isMavlink()) {
```

Comment:
I think it might be nice to do this for more than just mavlink, but we can revisit that later.